e2e_playwright/st_plotly_chart_select.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+import streamlit as st
+
+# Explicitly seed the RNG for deterministic results
+np.random.seed(0)
+
+df_bubble = px.data.gapminder()
+fig_bubble = px.scatter(
+    df_bubble.query("year==2007"),
+    x="gdpPercap",
+    y="lifeExp",
+    size="pop",
+    color="continent",
+    hover_name="country",
+    log_x=True,
+    size_max=60,
+)
+fig_bubble.update_layout(dragmode="select")
```

Comment:
I think we should remove most of the update_layout dragmode in this test since it is anyways determined automatically and only add it maybe to one test (e.g. by setting a box selection to dragmode pan) to make sure that the user can overwrite this.

```suggestion
```